tools/generators/pbxnativetargets/src/Generator/CalculateSharedBuildSettings.swift

```diff
@@ -99,15 +99,17 @@ extension Generator.CalculateSharedBuildSettings {
                     value: label.description.pbxProjEscaped
                 )
             )
-        } else {
-            // Used to work around CODE_SIGNING_ENABLED = YES for resource
-            // bundles in Xcode 14+
-            buildSettings.append(
-                .init(key: "CODE_SIGNING_ALLOWED", value: "NO")
-            )
         }
 
-        if productType == .uiTestBundle {
+        if productType == .framework {
```

Comment:
Just to clarify. You mentioned tvOS but IIUC this is being applied to any platform correct?

Btw, TIL that code signing was needed 🤔